Radio.net/dist/metadata.json

```diff
@@ -1,38 +1,38 @@
-{
-  "author": {
-    "name": "ACertainCoder",
-    "id": "320507572455079949"
-  },
-  "service": "Radio.net",
-  "description": {
-    "en": "Radio.net brings together more than 30,000 radio stations and podcasts. It has 4 million monthly users worldwide and has been downloaded more than 20 million times.",
-    "de": "Radio.net bietet Zugang zu mehr als 30.000 Sender und Podcasts und hat weltweit 4 Millionen monatliche Nutzer und über 24 Millionen App Downloads.",
-    "zh_CN": "Radio.net汇集了30,000多个广播电台和播客。 它在全球每月有400万用户，已被下载超过2000万次。"
-  },
-  "url": [
-    "www.radio.net",
-    "www.radio.de",
-    "www.radio.at",
-    "www.radio.fr",
-    "www.radio.se",
-    "www.radio.dk",
-    "www.radio.pt",
-    "www.radio.es",
-    "www.radio.it",
-    "www.radio.pl",
-    "br.radio.net",
-    "mx.radio.net",
-    "co.radio.net"
-  ],
-  "version": "2.2",
-  "logo": "https://i.imgur.com/3IBLlpc.png",
-  "thumbnail": "https://i.imgur.com/x2U3j65.png",
-  "color": "#67e350",
-  "tags": [
-    "radio",
-    "podcast",
-    "music",
-    "stream"
-  ],
-  "category": "music"
+{
+  "author": {
+    "name": "ACertainCoder",
+    "id": "320507572455079949"
+  },
+  "service": "Radio.net",
+  "description": {
+    "en": "Radio.net brings together more than 30,000 radio stations and podcasts. It has 4 million monthly users worldwide and has been downloaded more than 20 million times.",
+    "de": "Radio.net bietet Zugang zu mehr als 30.000 Sender und Podcasts und hat weltweit 4 Millionen monatliche Nutzer und über 24 Millionen App Downloads.",
+    "zh_CN": "Radio.net汇集了30,000多个广播电台和播客。 它在全球每月有400万用户，已被下载超过2000万次。"
+  },
+  "url": [
+    "www.radio.net",
+    "www.radio.de",
+    "www.radio.at",
+    "www.radio.fr",
+    "www.radio.se",
+    "www.radio.dk",
+    "www.radio.pt",
+    "www.radio.es",
+    "www.radio.it",
+    "www.radio.pl",
+    "br.radio.net",
+    "mx.radio.net",
+    "co.radio.net"
+  ],
+  "version": "2.2",

```

Comment:
maybe use regex? 
And you need to update the version

Reply:
Oh, yes no problem. I haven't seen that there was a version bump.